config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  devise_for :users
+
+  devise_for :users, :controllers => { :invitations => 'users_controller/invitations' }
```

Comment:
Aye copy paste from documentation.

We kinda talked a bit about this last week, I run a linter on my code that dislikes `:` because its not backwards compatible. I don't see the problem in using it if it isn't deprecated and it has backwards compatibility which new syntax doesn't. Its a supported feature of the language.